fullscreen/api/element-request-fullscreen-active-document.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<title>Element#requestFullscreen() when the document is not the active document</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<iframe allowfullscreen></iframe>
+<script>
+var t = async_test();
+
+onload = t.step_func(() => {
+  var iframe = document.querySelector("iframe");
+  var documentBeforeNav = iframe.contentDocument;
+  documentBeforeNav.onfullscreenchange = t.unreached_func('fullscreenchange event');
+  documentBeforeNav.onfullscreenerror = t.unreached_func('fullscreenerror event');
+
+  iframe.onload = t.step_func(() => {
+    var p = documentBeforeNav.documentElement.requestFullscreen();
+    // If a promise was returned, it should already be rejected, so its reject
+    // callback should be invoked before a second promise's callback.
+    // This test does not fail if promises aren't implemented.
+    if (p) {
+      var rejected = false;
+      p.catch(t.step_func(() => rejected = true));
+      Promise.resolve().then(t.step_func(() => assert_true(rejected)));
+    }
+    // Wait two animation frames to ensure that no events are fired. (One should
+    // be enough per spec, but this makes the test flaky in Firefox.)
```

Comment:
If you are concerned about the fullscreen transition... probably you can wait for a second instead I guess.